indico/modules/rb_new/schemas.py

```diff
@@ -77,6 +77,25 @@ class Meta:
         fields = ('start_dt', 'end_dt', 'is_valid', 'reservation')
 
 
+class ReservationDetailsOccurrenceSchema(mm.ModelSchema):
+    reservation = Nested(ReservationSchema)
+
+    class Meta:
+        model = ReservationOccurrence
+        fields = ('start_dt', 'end_dt', 'is_valid')
+
+
+class ReservationDetailsSchema(mm.ModelSchema):
+    room = Nested(RoomSchema)
+    booked_for_user = Nested(UserSchema, only=('full_name', 'phone', 'email'))
+    created_by_user = Nested(UserSchema, only=('full_name'))
+
+    class Meta:
+        model = Reservation
+        fields = ('start_dt', 'end_dt', 'repetition', 'booking_reason', 'created_dt', 'status_string',
+                  'uses_vc', 'needs_vc_assistance', 'booked_for_user', 'room', 'created_by_user', )
```

Comment:
let's omit things like `status_string` and `uses_*` as they'll likely go away. also, no need to serialize the room again. having the ID is enough (you can get the room data from the redux state)